cmd/kubeadm/app/api/install/install.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
What does registering an API group provide? I only need versioning, defaulting and documentation.